crates/runtime/src/embeddings/index/snapshots/runtime__embeddings__index__tests__scan_table_join_for_projection.snap

```diff
@@ -1,20 +1,22 @@
 ---
 source: crates/runtime/src/embeddings/index/mod.rs
+assertion_line: 476
```

Comment:
what's this snapshot for now?

Am i right in saying this was from the old part of this diff https://github.com/spiceai/spiceai/pull/6714/files#diff-82a945e65f7f9996abccad0124c5148de85fccae496e41c92c76448c8f033b81R501

Just don't want excess snapshots laying around.